keps/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -173,10 +167,9 @@ Metadata items:
   * Must be one of `Draft`, `Deferred`, `Approved`, `Rejected`, `Withdrawn`,
     `Final`, `Replaced`.
 * **authors** Required
-  * A list of authors for the KEP.  We require a name (which can be a psuedonym)
-    along with a github ID.  Other ways to contact the author is strongly
-    encouraged.  This is a list of maps.  Subkeys of each item: `name`,
-    `github`, `email` (optional), `slack` (optional).
+  * A list of authors for the KEP.
```

Comment:
seems fine for now, but in general we probably should work towards some standard contact information for KEPs and test owners

Reply:
yeah -- agreed.  As things stand it was just too verbose.